test/e2e/e2eutil/wait_util.go

```diff
@@ -241,9 +241,12 @@ func waitPodsDeleted(kubecli kubernetes.Interface, namespace string, timeout tim
 	return pods, err
 }
 
-// WaitUntilPodReady will wait until the first pod selected by the label 'lo' is ready. So this should be called for lablels that only select one pod.
-func WaitUntilPodReady(kubecli kubernetes.Interface, namespace string, lo metav1.ListOptions, timeout time.Duration) error {
+// WaitUntilOperatorReady will wait until the first pod selected for the label name=etcd-operator is ready.
+func WaitUntilOperatorReady(kubecli kubernetes.Interface, namespace string, timeout time.Duration) error {
```

Comment:
Let's just make it a method of upgrade test framework. It's only used there right now